src/core/resources/transactions/transaction.ts

```diff
@@ -63,14 +64,22 @@ export const fetchTransaction = async ({
   }
 
   try {
-    const response = await addysHttp.get<{
-      payload: { transaction: TransactionApiResponse };
-      meta: { status: string };
-    }>(`/${chainId}/${address}/transactions/${hash}`, {
-      params: { currency: currency.toLowerCase() },
-    });
-    const tx = response.data.payload.transaction;
-    if (response.data.meta.status === 'pending') {
+    const response =
+      await platformHttp.get<PlatformGetTransactionByHashResponse>(
+        PLATFORM_TRANSACTION_BY_HASH_PATH,
+        {
+          params: {
+            currency: currency.toLowerCase(),
+            address,
+            chainIds: chainId.toString(),
```

Comment:
Should validate these fetcher params against `useNetworkStore`'s `getSupportedTransactionsChainIds` list. We'll need to migrate this getter once the backend-driven networks config file breaks out the platform API vs the legacy APIs, but I believe backend has these matching right now